### PR TITLE
sets HeaderBeacon to null by default

### PR DESCRIPTION
**Pull request by aarongranick-okta**
## Description:

OIE/V2

Currently a "ghost" image is showing on the identify screen. This PR removes the HeaderBeacon unless an icon has been set.

### Issue:

- [OKTA-372808](https://oktainc.atlassian.net/browse/OKTA-372808)

### CODE REVIEW

**Comment by aarongranick-okta** [comment]
Screen Shot 2021-03-01 at 3 07 35 PM

**Review by nikhilvenkatraman-okta — APPROVED** [review]
Looks good, can we add a screenshot of the Identifier view without the beacon with the PR and get a 👍  from UX ?

**Comment by aarongranick-okta** [comment]
Before

Screen Shot 2021-03-01 at 3 51 41 PM

After

Screen Shot 2021-03-01 at 3 51 22 PM

**Review by haishengwu-okta — APPROVED** [review]

**Comment by sherizada-okta** [comment]
Looks like there's still padding for the security image. 

The header should look like this if there's no security image:
Screenshot 2021-03-01 at 4 17 17 PM

**Comment by aarongranick-okta** [comment]
Enroll view, has icon
Screen Shot 2021-03-01 at 4 19 12 PM

**Comment by aarongranick-okta** [comment]
Now removing header entirely from Identifier view:
@sherizada-okta 
Screen Shot 2021-03-01 at 4 23 09 PM

**Review by sherizada-okta — APPROVED** [review]

File: src/v2/view-builder/internals/BaseHeader.js [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
-    this.add(this.HeaderBeacon);
+    if (this.HeaderBeacon) {
+      this.add(this.HeaderBeacon);
+    }
   },
 
   postRender () {
-    $('#okta-sign-in').removeClass('no-beacon');
+    if (this.HeaderBeacon) {
+      $('#okta-sign-in').removeClass('no-beacon');
```
Comment (haishengwu-okta) [inline src/v2/view-builder/internals/BaseHeader.js:15]: use `util/Enum.WIDGET_CONTAINER_ID`

**Review by haishengwu-okta — APPROVED** [review]
One question and 🚀

**Comment by vincentvoong-okta** [comment]
@aarongranick-okta This looks great! A nit, the line that spans the width of the widget, and bisecting the beacon is off-center for authenticator pages. Can this be centered?

**Pull request closed without merging**